### PR TITLE
Device attributes

### PR DESCRIPTION
**Pull request by bielikb**
# Description
* Lists all audio devices
* Sets volume for recording and playback devices
* marks recording and playback devices as default
* muting logic

# Changes
Updated the view models with respective logic

Some of the logic could be extracted from the view models later on

### CODE REVIEW

**Comment by bielikb** [comment]
Resolves #5

File: sshh/ViewModels/ContentViewViewModel.swift [hunk 1]
```diff
@@ -17,8 +17,10 @@ final class ContentViewViewModel: ObservableObject {
     }
 
     func reload() {
-        inputViewModels = AudioDevice.allInputDevices().map {
-            InputRowViewModel(device: $0)
+        inputViewModels = AudioDevice.allDevices()
```
Comment (giacomorebonato) [inline sshh/ViewModels/ContentViewViewModel.swift:20]: The problem with this change is that now it's listing all the audio devices (also speakers), while this app focus is only on input devices.
It is also probably creating some problems for audio devices that are both input and output (webcams) and when selecting them I want to make sure that my following changes affect only the input signal.

Screen Shot 2020-10-13 at 1 49 14 PM

**Comment by giacomorebonato** [comment]
While we still need a better UI it looks more nicer to me to have in bold text the current default device, the logic seems to do the opposite at the moment.
You're welcome if you have better idea about this, we can still ask for help to some good designer.

Screen Shot 2020-10-13 at 1 54 28 PM

**Comment by bielikb** [comment]
I want to get the functionality right first, the UI/UX can be improved on top of that.
We can sync on your preferred use-case here :) 
I though you want to manage all devices ;) 

Btw, `AudioDevice.allInputDevices()` will pick also IO devices

**Comment by giacomorebonato** [comment]
Sorry Boris, I missed your last message!

**Pull request merged**